packages/onboarding/src/flow-progress/use-flow-progress.ts

```diff
@@ -50,6 +50,12 @@ const flows: Record< string, { [ step: string ]: number } > = {
 		processing: 6,
 		launchpad: 7,
 	},
+	sensei: {
```

Comment:
Surprised not to see `user` step here.

Reply:
I tried adding user but as we have too many steps it seems like the senseiSetup has almost half the bar completed which didn't look very nice.